README.md

```diff
@@ -19,6 +19,9 @@ Work in progress to adapt device specific and customer-specific out-of-tree conf
 While in build directory, run
 
     $ nix-build -I nixpkgs=nixpkgs -I spectrum-config=build-configurations/<config-name>.nix build-configurations/release/live/
+Currently available config files are:
+* imx8qm-config.nix
+* x86-nuc-config.nix
 
```

Comment:
Later this same README.md lists "imx8qm-evk" as the only supported device. Need update too?

Reply:
Right! Addressed.